packages/cli/src/commands/init/banner.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import chalk from 'chalk';
+
+const reactLogoArray = [
+  '                                                                ',
+  '                                                                ',
+  '                                                                ',
+  '                  ######                ######                  ',
+  '                ###     ####        ####     ###                ',
+  '               ##          ###    ###          ##               ',
+  '               ##             ####             ##               ',
+  '               ##             ####             ##               ',
+  '               ##           ##    ##           ##               ',
+  '               ##         ###      ###         ##               ',
+  '                ##  ########################  ##                ',
+  '             ######    ###            ###    ######             ',
+  '         ###     ##    ##              ##    ##     ###         ',
+  '      ###         ## ###      ####      ### ##         ###      ',
+  '     ##           ####      ########      ####           ##     ',
+  '    ##             ###     ##########     ###             ##    ',
+  '     ##           ####      ########      ####           ##     ',
+  '      ###         ## ###      ####      ### ##         ###      ',
+  '         ###     ##    ##              ##    ##     ###         ',
+  '             ######    ###            ###    ######             ',
+  '                ##  ########################  ##                ',
+  '               ##         ###      ###         ##               ',
+  '               ##           ##    ##           ##               ',
+  '               ##             ####             ##               ',
+  '               ##             ####             ##               ',
+  '               ##          ###    ###          ##               ',
+  '                ###     ####        ####     ###                ',
+  '                  ######                ######                  ',
+  '                                                                ',
+  '                                                                ',
+  '                                                                ',
+  '                                                                ',
+];
+
+const welcomeMessage = '                   Welcome to React Native!';
+const heyThere =
+  'Hey there! I am going to initialize a fresh react-native project: ';
```

Comment:
Maybe we don't need a "hey message", but rather a "Learn once, write everywhere" tagline? :)